api/v1/middleware/auth.js

```diff
@@ -13,7 +14,11 @@ module.exports = function (req, res, next) {
 
 	return AuthService.verify(auth)
 		.then(function (decoded) {
-			req.auth = decoded;
+			req.auth = true;
```

Comment:
I kept it in there because `!req.auth` could mean something different than `!req.user`. As in, lacking a valid auth token is potentially different than lacking a user from that token. It's also only an extra key-value pair with a bool as the value.